api/v1/submissions/SubmissionHandler.inc.php

```diff
@@ -0,0 +1,142 @@
+<?php
+
+/**
+ * @file api/v1/submission/SubmissionHandler.inc.php
+ *
+ * Copyright (c) 2014-2016 Simon Fraser University Library
+ * Copyright (c) 2003-2016 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * @class SubmissionHandler
+ * @ingroup api_v1_submission
+ *
+ * @brief Handle API requests for submission operations.
+ *
+ */
+
+import('lib.pkp.classes.handler.APIHandler');
+
+class SubmissionHandler extends APIHandler {
+
+	/**
+	 * The unique endpoint string for this handler
+	 *
+	 * @param string
+	 */
+	protected $_handlerPath = 'submissions';
```

Comment:
Ok, I addressed the issue with not re-declaring the `$_handlerPath` property in this commit: https://github.com/NateWr/ojs/commit/ab6df15eca8fa945079b8ca70f1b6da2d50929d6

However, I've left the check and exception code out. It looks like you've got a pattern going for throwing exceptions in your branch. None of that code is here, so I think it makes most sense to leave this for you to implement once the code is merged. Does that sound alright?